targeted/dns-email-spoofing.js

```diff
@@ -0,0 +1,188 @@
+/**
+ * Contributed by Astra Security (https://www.getastra.com/)
+ * @author Prince Mendiratta <prince.mendiratta@getastra.com>
+ */
+
+var pluginid = 100031;
+var providerAddress = "dns.google";
+
+var URI = Java.type("org.apache.commons.httpclient.URI")
+var HttpSender = Java.type("org.parosproxy.paros.network.HttpSender")
+var Model = Java.type("org.parosproxy.paros.model.Model")
+var HistoryReference = Java.type("org.parosproxy.paros.model.HistoryReference")
+var Control = Java.type("org.parosproxy.paros.control.Control")
+var ExtensionAlert = Java.type("org.zaproxy.zap.extension.alert.ExtensionAlert")
+var Alert = Java.type ("org.parosproxy.paros.core.scanner.Alert")
+
+var session = Model.getSingleton().getSession();
+var connectionParams = Model.getSingleton().getOptionsParam().getConnectionParam();
+var extLoaderAlert = Control.getSingleton().getExtensionLoader()
+
+// Print statements using script name
+function logger() {
+    print("[" + this["zap.script.name"] + "] " + arguments[0]);
+}
+
+/**
+ * Check for SPF / DMARC policies on a website.
+ * 
+ * A function which will be invoked against a specific "targeted" message.
+ * 
+ * @param msg - the HTTP message being acted upon. This is an HttpMessage object.
+ */
+function invokeWith(msg) {
+
+    var url = msg.getRequestHeader().getURI().toString();
+    // To check if script is running
+	logger("Testing script against URL - " + url);
```

Comment:
Throughout the script could you make the indents consistent, there are other really noticeable occurrences in fetchRecords as well, likely others as well.

Reply:
Updated.